Content.Shared/Atmos/EntitySystems/SharedGasTileOverlaySystem.cs

```diff
@@ -72,14 +77,18 @@ public static Vector2i GetGasChunkIndices(Vector2i indices)
             [ViewVariables]
             public readonly byte[] Opacity;
 
+            [ViewVariables]
+            public readonly float Temperature;
+
```

Comment:
Can you split this out to its own data? GasOverlayData is already super expensive to network because it pretty much gets sent every atmos tick and sending an entire float guarantees it.

Reply:
I'll look into it later

Reply:
Also look at `CheckTemperatureTolerance`, a tile is only dirtied due to temp if it changes by enough to change the distortion by 5%, or if the distortion becomes 0 or rises above 0. It's not dirtying everywhere due to every little fluctuation.

Reply:
Kicking this can down the road as per discord